src/main/scala/beam/agentsim/events/PathTraversalEvent.scala

```diff
@@ -215,7 +215,7 @@ object PathTraversalEvent {
     val arrivalTime: Int = attr(ATTRIBUTE_ARRIVAL_TIME).toInt
     val mode: BeamMode = BeamMode.fromString(attr(ATTRIBUTE_MODE)).get
     val legLength: Double = attr(ATTRIBUTE_LENGTH).toDouble
-    val linkIdsAsStr = attr(ATTRIBUTE_LINK_IDS)
+    val linkIdsAsStr = Option(attr(ATTRIBUTE_LINK_IDS)).getOrElse("")
```

Comment:
What's the point of this? To handle a null from `attr`? If so wouldn't it be better to change `attr` or just do a null check?

Reply:
Yes, it is to make sure that there will be no NPE. It's common way to me to interact with java code when `null` is possible by just wrapping it to `Option`. I thought it's common knowledge. I can add comments why it's done this way

Reply:
Nah - I'm aware of it - just wasn't sure if we wanted to make changes deeper. I'm fine with this though :)